### ISSUE TITLE
Add config-file-validator to Fedora

### ISSUE THREAD

**Issue by kehoecj** [Boeing/config-file-validator#43]
Build an RPM for the config-file-validator and [go through the process](https://docs.fedoraproject.org/en-US/package-maintainers/New_Package_Process_for_New_Contributors/) of getting it added to Fedora. After it's been added to Fedora it can be [added to EPEL](https://docs.fedoraproject.org/en-US/epel/epel-package-request/)

**Comment by kranurag7** [comment]
@kehoecj Thanks for the validator, I am looking forward to use this project for linting YAMLs. 

I released a minor version of config-file-validator on a fork [here](https://github.com/kranurag78/config-file-validator/releases/tag/v0.0.8) 

It uses [goreleaser](https://goreleaser.com/) to build [tarballs, debian and rpm packages](https://github.com/kranurag78/config-file-validator/releases/tag/v0.0.8). 
It uses goreleaser integration with [ko](https://ko.build) to build [container images](https://github.com/kranurag78/config-file-validator/pkgs/container/validator) for arm64 and amd64 platform. 

- The release config can be found [here](https://github.com/kranurag78/config-file-validator/blob/main/.github/workflows/goreleaser.yml)
- The goreleaser config can be found [here](https://github.com/kranurag78/config-file-validator/blob/main/.goreleaser.yaml) 

Let me know if there are any comments on this and if you want to proceed ahead with goreleaser. I'll create a PR afterward.

**Comment by kehoecj** [comment]
@kranurag7 Thanks for taking a look at this - we're using go-release-action now (thanks to @LeslieLeung). Would this be able to integrate with that to produce the RPM's and DEB's?

**Comment by kranurag7** [comment]
@kehoecj I think we cannot de that with go-release-action. It seems they only [support](https://github.com/wangyoucao577/go-release-action#:~:text=Publish%20.zip%20for%20windows%20and%20.tar.gz%20for%20Unix%2Dlike%20OS%20by%20default%2C%20optionally%20to%20disable%20the%20compression.) releasing binaries wrapped in `.zip` or tarballs.

**Comment by kehoecj** [comment]
@kranurag7 does goreleaser support our current releases (windows/Linux/macOS binaries) in addition to .deb and .rpm or can we use both goreleaser and go-release-action together in separate stages?

**Comment by kranurag7** [comment]
Yes, goreleaser do support [binaries including debs and rpms](https://github.com/kranurag78/config-file-validator/releases/tag/v0.0.8). 

Regarding using together, I think it'll be good if only one of the job having `contents: write` permissions writes to the releases page. But theoretically, yes, it seems doable where we will have one job using go-release-action and another job running goreleaser.

**Comment by kehoecj** [comment]
Okay great - sounds like we can just go with goreleaser then. If you can modify what you have on your fork to include the output of the current release job then I think that will be ready for submission
